enumerable.rb

```diff
@@ -0,0 +1,233 @@
+module Enumerable
+  def my_each
+    return to_enum unless block_given?
```

Comment:
* [ ] Improve this by passing the name of the method as an argument to ```to_enum(my_each)```.```return to_enum(my_each) unless block_given?``` instead of ```return to_enum unless block_given?```

Reply:
changes are done.